CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
-## [0.3.0-dev] - 2021-01-28
+## [0.3.0] - 2021-02-23
 ### Added
-  * TBD
+  * Secured system container initial mounts (mount/remount/unmounts on these from within the container are now restricted). See [here](docs/user-guide/security.md#initial-mount-immutability) for details.
```

Comment:
* Added support and required automation for Sysbox-in-Docker deployments.

Reply:
Mmm ... not sure this belongs in the changelog, since Sysbox-in-Docker is not technically part of Sysbox. For this same reason we don't mention changes in the Sysbox test infra in the changelog.

Reply:
I see your point, but testing enhancement is typically not something that an end-user can benefit from, Sysbox-in-Docker on the other hand offers an interesting deployment alternative for special scenarios. IMO, the fact that we have validated this scenario is something end-users should know about when they look at our repo's changelog.

Reply:
Ok done.